fe/fe-core/src/main/java/com/starrocks/server/OlapTableFactory.java

```diff
@@ -101,7 +102,12 @@ public Table createTable(LocalMetastore metastore, Database db, CreateTableStmt
         ColocateTableIndex colocateTableIndex = metastore.getColocateTableIndex();
         String tableName = stmt.getTableName();
 
-        LOG.debug("begin create olap table: {}", tableName);
+        if (stmt instanceof CreateTemporaryTableStmt) {
```

Comment:
i don't think these debug log is necessary to commit

Reply:
> i don't think these debug log is necessary to commit

This log existed before. It only distinguishes temporary tables in my change. I think it is acceptable to retain it, won't be a big problem